pipelines/tasks/hisat2.wdl

```diff
@@ -16,7 +16,10 @@ task HISAT2PE {
       --rg PL:ILLUMINA --rg PU:${sample_name} \
       --new-summary --summary-file ${output_name}.log \
       --met-file ${output_name}.hisat2.met.txt --met 5 \
-      -p 4 -S ${output_name}.sam
+      --seed 12345 \
+      -k 10 \
+      --secondary \
+      -p 4 -S ${output_name}.sam 
```

Comment:
And do we want to have comments about parameter choices next to the parameter inline instead of above? Just a thought